js/Source/Charts/Chart.js

```diff
@@ -279,6 +279,7 @@ Chart = new Class({
     var v2yAxis = options.yAxis;
     var size = this.getSizeOptions();
     var self = this;
+    if (options.columnGrouping) this.options.columnGrouping = options.columnGrouping;
```

Comment:
Just confirming, this allows you to specify column grouping in the response (in addition to the options passed in at instantiation), yes? That works.